config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   RANDOM_ID: process.env.randomId || '',
   EVENTS_ID: process.env.eventsId || '',
   ICON_URL: process.env.iconUrl || '',
+  API_AI_TOKEN: process.env.API_AI_TOKEN || 'pass your own key at runtime',
```

Comment:
I discussed using https://www.npmjs.com/package/dotenv for this with @oddoye-david